master/reference/public-cloud/aws.md

```diff
@@ -0,0 +1,152 @@
+---
+title: AWS
+---
+
+Calico is designed to provide high performance massively scalable virtual
+networking not only for private data centers, but also within a
+public cloud such as Amazon Web Services (AWS).  The following information
+details necessary modifications to your standard AWS deployment to allow
+Calico to function successfully.
+
+
+## Prerequisite: Host Connectivity
+
+For Calico to run successfully on AWS, we must first ensure
+each compute node has network connectivity with each other. Once this requirement
+is met, each host should display a established BGP
+session using calicoctl:
+
+```
+$ sudo calicoctl node status
+Calico process is running.
+
+IPv4 BGP status
++--------------+-------------------+-------+----------+-------------+
+| PEER ADDRESS |     PEER TYPE     | STATE |  SINCE   |    INFO     |
++--------------+-------------------+-------+----------+-------------+
+| 172.17.8.101 | node-to-node mesh | up    | 17:19:54 | Established |
++--------------+-------------------+-------+----------+-------------+
+
+IPv6 BGP status
+No IPv6 peers found.
+```
+
+## Allowing Workload-to-Workload Traffic
+
+By default, each EC2 instance performs source/destination checks on inbound and outbound traffic.
+This means that the instance must be the source or destination of any traffic it
+sends or receives.
+
+Since Calico allocates a unique IP to each container, container traffic leaving
+and entering the EC2 instance will have an "unrecognized" source address
+and traffic will be dropped.
+
+This behavior can be disabled for inter-VPC workloads,
+and can be circumvented for cross-VPC workloads.
+
+#### Inter-VPC: Disable Src/Dest Checks
+
+See the [AWS guide on disabling src/dest checks for an EC2 instance](http://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/VPC_NAT_Instance.html#EIP_Disable_SrcDestCheck).
+
+#### Cross-VPC: Enable IP-in-IP
+
+Unfortunately, cross-VPC src/dest checks cannot be disabled in AWS.
```

Comment:
This isn't quite right.  The src/dst check is just a per-node RPF check.

Cross-subnet traffic actually goes through a router, which isn't aware of pod traffic, and so tunneling is required.